herald_rules/add_assignee_for_workflows.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "This is a rule to add @gagoar as a assignee when workflow changes",
-  "includes": ".github/workflows/*",
+  "includes": "src/*.ts",
```

Comment:
why doesn't this listen to workflow changes anymore?

Reply:
this is just a tests, I'm having an issue with rules matching always, playing with it right now.

Reply:
I will leave it with this change, so we can always be sure, that assignee is not applied other than src/*.ts is added.